demo/rviz.launch.py

```diff
@@ -0,0 +1,60 @@
+from launch import LaunchDescription
+from launch_ros.actions import Node, PushRosNamespace
+from launch.actions import DeclareLaunchArgument, OpaqueFunction
+from launch.substitutions import EnvironmentVariable, LaunchConfiguration
+
+def replace_rviz_fixed_frame(input_file, output_file, fix_frame):
+    with open(input_file, 'r') as file:
+        rviz_content = file.read()
+
+    rviz_content = rviz_content.replace('Fixed Frame: camera_link', f'Fixed Frame: {fix_frame}_link')
+
+    with open(output_file, 'w') as file:
+        file.write(rviz_content)
+
+def launch_setup(context, *args, **kwargs):
+    robot_namespace = LaunchConfiguration("robot_namespace").perform(context)
+    sensor_namespace = LaunchConfiguration("sensor_namespace").perform(context)
+
+    default_rviz_conf = "/root/.rviz2/default.rviz"
+    rviz_conf = "/root/.rviz2/modified_default.rviz"
+    replace_rviz_fixed_frame(default_rviz_conf, rviz_conf, f'{sensor_namespace}_link')
+
+    remapping = []
+    if robot_namespace:
+        remapping.append(('/clicked_point', f'/{robot_namespace}/clicked_point'))
+        remapping.append(('/goal_pose', f'/{robot_namespace}/goal_pose'))
+        remapping.append(('/initialpose', f'/{robot_namespace}/initialpose'))
+        remapping.append(("/tf", f"/{robot_namespace}/tf"))
+        remapping.append(("/tf_static", f"/{robot_namespace}/tf_static"))
+
+    rviz = Node(
+        package='rviz2',
+        executable='rviz2',
+        name='rviz2',
+        namespace=sensor_namespace,
+        remappings=remapping,
+        arguments=['-d', rviz_conf],
+        output='screen',
+    )
+
+    return [PushRosNamespace(robot_namespace), rviz]
```

Comment:
What wold happen when `robot_namespace` = `robot1` and `sensor_namespace` = `sensor1`? Will it be `/robot1sensor1` or `/robot1/sensor1`?

Reply:
/robot1/sensor1